server/cmd/BUILD

```diff
@@ -9,10 +9,13 @@ go_binary(
     args = [
         "--config_file=config/buildbuddy.local.yaml",
         "--max_shutdown_duration=3s",
-    ],
```

Comment:
some changes to internal will probably also be needed

Reply:
Noted, I'll send another PR